mu/locale/fr/LC_MESSAGES/mu.po

```diff
@@ -584,579 +74,633 @@ msgstr ""
 
-#: mu/interface/main.py:68
+#: mu/interface/main.py:100
 msgid "Mode"
 msgstr "Mode"
 
-#: mu/interface/main.py:69
+#: mu/interface/main.py:101
 msgid "Change Mu's mode of behaviour."
 msgstr "Change le mode de Mu."
 
-#: mu/interface/main.py:71
+#: mu/interface/main.py:106
 msgid "New"
 msgstr "Nouveau"
 
-#: mu/interface/main.py:72
+#: mu/interface/main.py:107
 msgid "Create a new Python script."
 msgstr "Crée un nouveau script Python."
 
-#: mu/interface/main.py:73
+#: mu/interface/main.py:111
 msgid "Load"
 msgstr "Charger"
 
-#: mu/interface/main.py:74
+#: mu/interface/main.py:112
 msgid "Load a Python script."
 msgstr "Charge un script Python."
 
-#: mu/interface/main.py:75
+#: mu/interface/main.py:116
 msgid "Save"
 msgstr "Enregistrer"
 
-#: mu/interface/main.py:76
+#: mu/interface/main.py:117
 msgid "Save the current Python script."
 msgstr "Enregistre le script Python courant."
 
-#: mu/interface/main.py:85
+#: mu/interface/main.py:131
 msgid "Zoom-in"
 msgstr "Zoomer"
 
-#: mu/interface/main.py:86
+#: mu/interface/main.py:132
 msgid "Zoom in (to make the text bigger)."
 msgstr "Augmente le niveau de zoom (pour rendre le texte plus gros)."
 
-#: mu/interface/main.py:87
+#: mu/interface/main.py:136
 msgid "Zoom-out"
 msgstr "Dé-zoomer"
 
-#: mu/interface/main.py:88
+#: mu/interface/main.py:137
 msgid "Zoom out (to make the text smaller)."
 msgstr "Diminue le niveau de zoom (pour rendre le texte plus petit)."
 
-#: mu/interface/main.py:89
+#: mu/interface/main.py:141
 msgid "Theme"
 msgstr "Thème"
 
-#: mu/interface/main.py:90
+#: mu/interface/main.py:142
 msgid "Toggle theme between day, night or high contrast."
 msgstr "Change de thème entre jour, nuit ou contraste élevé"
 
-#: mu/interface/main.py:93
+#: mu/interface/main.py:149
 msgid "Check"
 msgstr "Vérifier"
 
-#: mu/interface/main.py:94
+#: mu/interface/main.py:150
 msgid "Check your code for mistakes."
 msgstr "Vérifie que le programme ne contient pas d'erreur."
 
-#: mu/interface/main.py:95
+#: mu/interface/main.py:155
+msgid "Tidy"
+msgstr "Nettoyer"
+
+#: mu/interface/main.py:156
+msgid "Tidy up the layout of your code."
+msgstr "Nettoyer l'arrangement de ton code."
+
+#: mu/interface/main.py:160
 msgid "Help"
 msgstr "Aide"
 
-#: mu/interface/main.py:96
+#: mu/interface/main.py:161
 msgid "Show help about Mu in a browser."
 msgstr "Montre l'aide à propos de Mu dans un navigateur."
 
-#: mu/interface/main.py:97
+#: mu/interface/main.py:165
 msgid "Quit"
 msgstr "Quitter"
 
-#: mu/interface/main.py:98
+#: mu/interface/main.py:165
 msgid "Quit Mu."
 msgstr "Quitte Mu."
 
-#: mu/interface/main.py:177
+#: mu/interface/main.py:263
+msgid "Close file"
+msgstr "Ferme le ficher"
+
+#: mu/interface/main.py:305
 msgid "Mu {}"
 msgstr "Mu {}"
 
-#: mu/interface/main.py:374
-msgid "Filesystem on micro:bit"
-msgstr "Système de fichier micro:bit"
+#: mu/interface/main.py:537
+msgid "Cannot connect to device on port {}"
+msgstr "Impossible de se connecter à un périphérique sur le port {}"
```

Comment:
Not 100% satisfied with device → périphérique, also not sure how best to translate board.